sources/packages/backend/libs/services/src/notifications/notification/notification-actions.service.ts

```diff
@@ -1199,6 +1200,41 @@ export class NotificationActionsService {
     );
   }
 
+  /**
+   * Create supporting user information notification for student.
+   * @param notification notification details.
+   * @param entityManager entity manager to execute in transaction.
+   */
+  async saveSupportingUserInformationNotification(
+    notification: SupportingUserInformationNotification,
+    entityManager: EntityManager,
+  ): Promise<void> {
+    const auditUser = this.systemUsersService.systemUser;
+    const { templateId } =
+      await this.notificationMessageService.getNotificationMessageDetails(
+        NotificationMessageType.SupportingUserInformationNotification,
+      );
+    const supportingUserInformationNotification = {
+      userId: auditUser.id,
```

Comment:
The userId must be still student userid. only line 1233 must be using audit user id.